Library/Homebrew/test/cask/cli/outdated_spec.rb

```diff
@@ -0,0 +1,88 @@
+describe Hbc::CLI::Outdated, :cask do
+  before do
+    shutup do
+      cask = Hbc::CaskLoader.load_from_file(TEST_FIXTURE_DIR/"cask/Casks/basic-cask.rb")
```

Comment:
Define these like `let(:basic_cask) { Hbc::CaskLoader.load_from_file(TEST_FIXTURE_DIR/"cask/Casks/basic-cask.rb") }` above the `before` block.